src/containers/BooksList.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { connect } from 'react-redux';
+import { removeBook, changeFilter } from '../actions';
+
+import CategoryFilter from '../components/CategoryFilter';
+import Book from '../components/Book';
+
+const getVisibleBooks = (books, currentFilter) => {
```

Comment:
With the version of `mapStateToProps` I suggested in `BooksForm.js` you won't need this `getVisibleBooks` function. Albeit if you want to use this to abstract the extraction of books based on filter value then this function should be in a `helpers` folder which can just be imported here (and it can just be used in that suggested version of `mapStateToProps`)